CMakeLists.txt

```diff
@@ -449,10 +449,10 @@ else()
   elseif(ARM7)
     message(STATUS "AES support not available on ARMv7")
   elseif(ARM8)
-    CHECK_CXX_ACCEPTS_FLAG("-march=${ARCH}+crypto" ARCH_PLUS_CRYPTO)
+    CHECK_CXX_ACCEPTS_FLAG("-march=${ARCH}+cncrypto" ARCH_PLUS_CRYPTO)
     if(ARCH_PLUS_CRYPTO)
       message(STATUS "Crypto extensions enabled for ARMv8")
-      set(ARCH_FLAG "-march=${ARCH}+crypto")
+      set(ARCH_FLAG "-march=${ARCH}+cncrypto")
```

Comment:
Those seem completely wrong. Are you sure about it or did you just run a batch replace ?